### PR TITLE
Added a11y fixes Ari developed to Riff Edu

### PR DESCRIPTION
**Pull request by iamtomhanks**
#### Summary
Copied a11y revisions from Ari's PRs on riff-rtc. See below.

https://github.com/rifflearning/riff-rtc/pull/27
https://github.com/rifflearning/riff-rtc/pull/30

#### Ticket Link
https://trello.com/c/elxfxgiX/211-add-a11y-fixes-ari-is-developing

#### Checklist
- [ ] Ran `make check-style` to check for style errors (required for all pull requests)
- [ ] Ran `make test` to ensure unit and component tests passed
- [x] Has UI changes
- [ ] Includes text changes and localization file ([.../i18n/en.json](https://github.com/mattermost/mattermost-webapp/blob/master/i18n/en.json)) updates
- [ ] Touches critical sections of the codebase (auth, posting, etc.)

### CODE REVIEW

**Comment by mlippert** [comment]
[Add a11y fixes Ari is developing](https://trello.com/c/elxfxgiX/211-add-a11y-fixes-ari-is-developing)

**Comment by iamtomhanks** [comment]
Alright I believe I've addressed those issues.

**Comment by iamtomhanks** [comment]
@mlippert Ah, sorry Mike, missed that.

File: components/dashboard/components/ChartTable.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+import styled from 'styled-components';
+
+ const SROnly = styled.div`
```
Comment (mlippert) [inline components/dashboard/components/ChartTable.js]: I assume the one space indent is a typo?

File: components/dashboard/components/ChartTable.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+position:absolute;
+left:-10000px;
+top:auto;
+width:1px;
+height:1px;
+overflow:hidden;
+`;
+
+ const ChartTable = ({cols, rows}) => (rows.length) ? (
```
Comment (mlippert) [inline components/dashboard/components/ChartTable.js]: I assume the one space indent is a typo?

**Comment by iamtomhanks** [comment]
Sorry, the erroneous spaces have been removed.

**Pull request merged**